app/assets/javascripts/fae/form/_filtering.js

```diff
@@ -31,17 +31,17 @@ Fae.form.filtering = {
     // hardcode _this because this === Fryr object
     var _this = Fae.form.filtering;
     var post_url = _this.$filter_form.attr('action');
-    var $results_table = ($(".js-results-table").length) ? $(".js-results-table").first() : _this.$filter_form.find('table');
- 
+    var $results_table = ($(".js-results-table").length) ? $(".js-results-table").first() : _this.$filter_form.next('table');
```

Comment:
@tshedor This is fixing an update that broke old implementations: https://github.com/wearefine/fae/pull/174/files#diff-cebcdb529c6537915f3dae2fe8212a79L40